### ISSUE TITLE
No icons in about:addons

### ISSUE THREAD

**Issue by tsjnachos117** [MoonchildProductions/UXP#1029]
That said, here's my bug: if I go to the about:addons page, none of my addon icons load, with puzzle pieces being displayed instead. This has been plaguing several of my systems since the WebExt API was purged (version 2019.02.11).

After doing some digging, I discovered that if I disabled Classic Theme Restorer, all was well again. When I re-enabled the addon, I found that disabling one option fixed the problem for me: go to "Add-on manager" > and uncheck the box that says "Add-on lists: show add-on version". Strangely, this does the opposite of what you might expect: addon version numbers are shown **after this option is disabled**. This doesn't affect Waterfox, so I'm guessing it's *NOT* a bug in the extension itself.

To summarize, Basilisk is doing something funky with this addon, and I think it's a bug in Basilsk itself.

To give a better idea of what the issue is, I've attached a picture below. (I'm not sure how two different items got selected at the same time, but whatever.)

Operating systems: Ubuntu, Fedora, Debian, and Windows Vista (the latter has an unofficial build). All X64 systems.

![Corruption](https://user-images.githubusercontent.com/6446848/55120621-aad47680-50ee-11e9-959a-1075ae6855ca.png)

**Comment by tsjnachos117** [comment]
Update: before typing this bug report, I updated to the newest version of Classic Theme Restorer. Or so I thought: turns out I had an old version. Consequently, after installing [a newer version](https://github.com/Aris-t2/ClassicThemeRestorer/releases), the problem went away. I guess this issue report can be closed.

**Comment by mattatobin** [comment]
Please use the [Pale Moon Forum](http://forum.palemoon.org/) for things like this BEFORE you open an issue here. You are creating too much noise on the issue tracker for non-issues.
